websocket-proxy/src/integration.rs

```diff
@@ -0,0 +1,308 @@
+mod test {
```

Comment:
Wdyt about moving integration tests to a `tests/` dir? This way the `integration` feature can be removed.

Reply:
Resolving this for now, this can be revisited in a follow up PR.